src/config.ts

```diff
@@ -43,3 +46,19 @@ export const showHomeServerDetail = loginScreen?.showHomeServerInfo ?? true;
 
 // Default footer links
 export const showDefaultFooterLinks = loginScreen?.showMatrixDefaultFooterLinks ?? true;
+
+// show/hide liberate your communication text
+export const showLiberateYourCommunicationText = authenticatedHomeScreen?.showLiberateYourCommunicationText ?? true;
+
+//show/hide welcome to {brand} text
+export const showWelcomeToElementText = authenticatedHomeScreen?.showWelcomeToElementText ?? true;
+
+//find logo secondary
+export const logoSecondary = logoConfig?.logo_secondary ?? null;
+
+
+// tab condition
+export const tabbedView =config['tabbedView'];
+export const tabbedViewdisplaySecondaryLogo  = tabbedView?.displaySecondaryLogo ?? false;
+export const logoUrlTabbedViewFooter =  tabbedView?.imgUrl;
```

Comment:
For consistency please rename 
imgUrl to secondaryLogoUrl
description to secondaryLogoAltText

Reply:
done